server/resp/src/main/java/org/infinispan/server/resp/Intrinsics.java

```diff
@@ -62,32 +78,40 @@ public static String bulkString(ByteBuf buf, Resp2LongProcessor longProcessor) {
       }
       // If we didn't read any bytes then the number wasn't parsed properly
       if (pos == buf.readerIndex()) {
-         return null;
+         return -1;
       }
       int size = (int) longSize;
+      if (size < 0) {
```

Comment:
Should this be `<=`? We are using this method before creating a `RespCommand`, I don't see a case with 0 size.

Reply:
You can have an argument with 0 bytes as per RESP protocol. Or maybe you are thinking of something else?